source/manual/publish-special-routes.html.md.erb

```diff
@@ -14,5 +14,14 @@ All new special routes should be created in this tool, and existing special rout
 
 There are two Jenkins jobs to publish special routes:
 
-- Publish a single special route: <%= RunJenkinsJob.terse_links("Publish_Single_Special_Route") %>
-- Publish all special routes: <%= RunJenkinsJob.terse_links("Publish_Special_Routes") %>
```

Comment:
Try [links](https://github.com/alphagov/govuk-developer-docs/blob/b2f0943051c484f49039fe76d3183db4c29f5b20/app/run_jenkins_job.rb#L4-L20) instead of terse_links if you want a list formatting.

Reply:
Ohhhh! Thanks

Reply:
I didn't even know `RunJenkinsJob` was a thing. 🤩